specification.md

```diff
@@ -0,0 +1,89 @@
+# OpenTelemetry/Prometheus Compatibility Specification
+
+Status: [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/document-status.md)
+
+## Abstract
+
+OpenTelemetry is aiming to provide compatibility with
+Prometheus and OpenMetrics. This document explains the
+extent of the support. OpenTelemetry collector and libraries
+will align with the compatibility requirements defined in
+this spec.
+
+## Goals
```

Comment:
I think in one of our first meetings there was the idea of "To the OpenTelemetry collector an OpenTelemetry target and a prometheus instrumented target is indistinguishable."

Reply:
Added this.